spec/swagger.yaml

```diff
@@ -2041,11 +2070,14 @@ paths:
 
   /context-api/v2/projects/{projectId}/bindings:
     post:
-      summary: Create bindings
+      summary: Create string to context bindings
       description: |-
-        Link context images with the strings they contain. The binding tells Smartling which context images to show translators for which strings.
-        A maximum of **150** bindings can be created per request. A request with more than **150** items will return a **400** response indicating validation error.
-        Basic bindings request should contain an object with "bindings" JSON array with at least one element (binding), featuring *contextUid* and *stringHashcode* properties:
+        Link context files with the strings they contain. 
+        The binding tells Smartling which context images to show translators for which strings.
+        A maximum of `150` bindings can be created per request. 
```

Comment:
I think this is kind of a limit too. Do we want to avoid declaring it here too?

Reply:
Where else would we describe this limit?